layouts/tags.liquid

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
If the user needs to prepare a list of tags, what's stopping them from just creating a separate placeholder page for each tag?

Reply:
Before addressing your comment, I made a typo and the pagination `permalink` should be `/notes/{{ tag }}/` as I don't think its necessary to include the placeholder text tag before the actual hashtag. 

Yeah I thought about that. This way the pagination creates the pages for each tag in `postTags`, but they most definitely could just create separate pages for each tag. I thought maybe pagination would be more convenient as it limits the amount of new files manually created by the user.

If the pagination data `postTags` could be another data source that updates as new tags are added within notes. That would help to make the process of creating tag pages more automated.

Reply:
Yeah I thought about that. This way the pagination creates the pages for each tag in `postTags`, but they most definitely could just create separate pages for each tag. I thought maybe pagination would be more convenient as it limits the amount of new files manually created by the user.

If the pagination data `postTags` could be another data source that updates as new tags are added within notes. That would help to make the process of creating tag pages more automated.

Reply:
That makes sense. I found an issue comment (https://github.com/11ty/eleventy/issues/1427#issuecomment-697336297) which should help us compute tags, in order to use them as a data source.

Reply:
Perfect. I had a look at the issue comment for computing tags inside a collection, it looks really helpful. I will attempt to prototype something up from the information provided in the comments.

Reply:
@tannerdolby you still interested in this?

Reply:
@binyamin hey! yes I'd like to try finishing this up. I apologize for the long period of time without correspondence. I had some job stuff going on and alot on my plate the last 2-3 weeks. I will do some work tonight + tomorrow and try getting the computed tags handled. Talk soon.